Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>this is very well written! Still, if you felt like playing with the groups by hand in code to show how the filters g^{-1}h actually behave and how they affect the inputs for example, I think it would make it even more accessible.</p><p> </p><p>On a high level, I'd try to build by hand the action of the GCNN on one or two inputs and show that the output is equivariant.</p><p> </p><p>Of course, only if you feel like doing it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks, I definitely think it would be informative to show that the output is invariant so I'll add that in.</p><p> </p><p>I took your advice and looked at the weights and there's basically no discernible structure. In fact their values are basically random normal. However, this did help me realize that G-CNN isn't initializing the weights as intended, so it was a worthwhile endeavor.</p>

Reply:
<p>I thought maybe W^{ij}_{identity} would be larger (in magnitude) on average but even that wasn't the case</p>